dbus/src/channel/ffichannel.rs

```diff
@@ -284,8 +284,14 @@ impl Channel {
     /// Removes a message from the incoming queue, or waits until timeout if the queue is empty.
     ///
     pub fn blocking_pop_message(&self, timeout: Duration) -> Result<Option<Message>, Error> {
```

Comment:
I thought I could modify blocking_pop_message, but I haven't checked if this is exported as API

Reply:
`Channel` trait's `blocking_pop_message` is a public API
https://docs.rs/dbus/latest/dbus/channel/struct.Channel.html#method.blocking_pop_message